app/webpacker/src/javascripts/commodities.js

```diff
@@ -752,6 +752,7 @@
                 showAllValues: true,
                 confirmOnBlur: true,
                 alwaysDisplayArrow: true,
+                displayMenu: 'overlay',
```

Comment:
Do we have any javascript tests for this? Is it appropriate to have coverage here?

Reply:
This is provided by the add-on so we can just assume they have throughly tested this feature

Reply:
Yeah, that's fair enough - this is essentially configuration